swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        self.age = age
```

Comment:
👍 Nice inclusion of an option `age` value.

Note that as implemented, any item added without an explicit age will be considered 0, or newest.